samples/matter/lock/README.rst

```diff
@@ -478,6 +510,54 @@ Complete the following steps to generate the Matter OTA combined image file:
 .. note::
     Keep the order in which the files are passed to the script, given that the Thread variant image file must be passed in front of the Wi-Fi variant image.
 
+Testing Lock Bluetooth LE Nordic UART Service
+====================================
+
+To test this feature, complete the following steps:
+
+#. Install `nRF Toolbox <https://www.nordicsemi.com/Products/Development-tools/nrf-toolbox>`_ on Android or iOS phone.
+
+#. Build the door lock application for Matter over Thread:
+
+   .. code-block:: console
+
+      west build -b nrf52840dk_nrf52840 -- -DCONFIG_CHIP_NUS=y 
```

Comment:
We should mention that nRF52840 is just an example here, or replace is with <your_board_name> like text.

Reply:
Is this extension only available to nRF52840 and nRF5340, not nRF21540 and nRF7002?